src/sass/cookie-policy.scss

```diff
@@ -1,5 +1,5 @@
 // Vanilla components
-@import 'vanilla-framework/scss/vanilla';
+@import 'vanilla-framework/scss/build';
```

Comment:
This includes a large amount of CSS. Please follow the guide @Caleb-Ellis mentioned in the vanilla issue.